src/pages/GovPages/Proposals/ProposalCard.tsx

```diff
@@ -0,0 +1,163 @@
+import React, { useState } from 'react'
+import styled from 'styled-components'
+import { Clock } from 'react-feather'
+import { Flex } from 'rebass'
+
+import { TYPE } from '../../../theme'
+import Row, { RowBetween } from '../../../components/Row'
+import { AutoColumn } from '../../../components/Column'
+import Card, { LightCard } from '../../../components/Card'
+
+import useInterval from '../../../hooks/useInterval'
+
+import HourGlass from '../../../assets/svg/hourglass.svg'
+import Block from '../../../assets/svg/block.svg'
+import Done from '../../../assets/svg/done_all.svg'
+
+import { VoteStatus, VoteStatusType, InProgress, Passed, Failed } from '../VoteStatus'
+
+const Container = styled(LightCard)<{ status: VoteStatusType }>`
+  padding: 1.25rem;
+  height: 88px;
+  position: relative;
+  cursor: pointer;
+  background: ${({ theme, status }) =>
+    status === InProgress
+      ? `linear-gradient(113.18deg, rgba(255, 255, 255, 0.35) -0.1%, rgba(0, 0, 0, 0) 98.9%), ${theme.dark1}`
+      : `linear-gradient(141.72deg, ${theme.bg1} -11.46%, ${
+          status === Passed ? 'rgba(5, 122, 85, 0.5)' : 'rgba(238, 46, 81, 0.5)'
+        } 180.17%)`};
+  background-blend-mode: overlay, normal;
+  border: none;
+
+  ::before {
```

Comment:
Is this used on hover or something? I've used before pseudoelements in the past to implement hovers with gradients, but I don't see any hover selector here (it might be in the parent component but I can't see it from here).

Reply:
This is for the gradient border of proposal card. 
Basically, it places a bigger container behind the element (top, left, bottom, right all -1px) with gradient background in order to make a gradient border of child component.

Reply:
Sounds good. I actually used a SVG for the border but this is better.